### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by stefanv**
I've updated the README to include more information on the date.

### CODE REVIEW

File: README.txt [hunk 1]
```diff
@@ -1 +1,3 @@
 Temporary repo for Git / GitHub training
+
+This is for 2016/09/28
```
Comment (stefanv) [inline README.txt]: This does look like today's date.  Did you mean to use international format, or should this be US date format?

Comment (tseclaudia) [reply README.txt]: Hi :)

Comment (sirlittle) [inline README.txt]: Maybe we should include the time as well?

File: README.txt [hunk 2]
```diff
@@ -1 +1,3 @@
 Temporary repo for Git / GitHub training
+
```
Comment (MichaelTamaki) [inline README.txt:2]: Hello!

File: README.txt (hunk 1 above)
Comment (JustinMi) [inline README.txt]: Hello

Comment (alvinzz) [reply README.txt]: Hi everyone!

File: README.txt (hunk 2 above)
Comment (wilson1yan) [inline README.txt:2]: Hello, this is good.

File: README.txt (hunk 1 above)
Comment (sonajeswani) [reply README.txt]: Hello!

**Pull request merged**